hr_employee_id/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+# © 2011, 2013 Michael Telahun Makonnen <mmakonnen@gmail.com>
+# © 2016 OpenSynergy Indonesia
+# © 2018 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Employee ID',
+    'version': '12.0.1.0.0',
+    'license': 'AGPL-3',
+    'category': 'Generic Modules/Human Resources',
+    'author':
+        'Brainbean Apps, '
+        'Michael Telahun Makonnen, '
+        'OpenSynergy Indonesia, '
+        'Camptocamp, '
+        'Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/hr',
+    'depends': [
+        'hr'
+    ],
+    'data': [
```

Comment:
`data` sequence as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L29)

Reply:
done